### PR TITLE
Gitignored swagger doc

### LINKED ISSUE #953: Position argument breaks swagger editor
Our current swagger documentation produces following errors:
grafik

So the position arguments should be removed. As this will result in inconsistent documents, they should be excluded from the version control, as they will be generated before deployment nevertheless.

### PR DESCRIPTION
**Pull request by JAmmermann-DLR**
- Added gitignore to swagger doc output folders
- Removed generated swagger docs from version control
- Removed position attributes from swagger annotations
- Minor improvements in the documentation
- Added jersey test framework to external lib (had to manually exclude junit and hamcrest dependencies)
- Refactored documentation resource (now mocked with the jersey test fw)
- Increased heap size of run configuration

Closes #953

### CODE REVIEW

**Review by maurpa — APPROVED** [review]

File: de.dlr.sc.virsat.server.test/src/de/dlr/sc/virsat/server/test/AJerseyTest.java [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+ * Test class wrapping the class from the Jersey Test Framework.
+ * Before and After have to be overwritten here for JUnit to work correctly.
+ * Also enables verbose debug output.
+ */
+public class AJerseyTest extends JerseyTest {
+	
+	@Before
+	public void setUp() throws Exception {
+        super.setUp();
+		enable(TestProperties.LOG_TRAFFIC);
```
Comment (SaMuellerDLR) [inline de.dlr.sc.virsat.server.test/src/de/dlr/sc/virsat/server/test/AJerseyTest.java:27]: Some weird whitespacing here.

**Review by SaMuellerDLR — CHANGES_REQUESTED** [review]
Nice work. Only super tiny thing.

File: de.dlr.sc.virsat.server.test/src/de/dlr/sc/virsat/server/test/AJerseyTest.java (hunk 1 above)
Comment (JAmmermann-DLR) [reply de.dlr.sc.virsat.server.test/src/de/dlr/sc/virsat/server/test/AJerseyTest.java:27]: Thanks, changed!

**Review by SaMuellerDLR — APPROVED** [review]
Approved!

**Pull request merged**